<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1393</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonSTLImport">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>STL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSphere">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Sphere</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Box</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCylinder">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cylinder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Geometry</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonInternal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Internal Flow</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonExternal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>External Flow</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxOnGround">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>On Ground</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Domain Size</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDomainAuto">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMaxZ">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMaxX">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMinY">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMaxY">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>max</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMinZ">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMinX">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBoundaryCondition">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Boundary Conditions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonControl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Simulation Controls</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNumerics">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Numerical Methods</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOpen">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>440</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSTLProperties">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>STL Properties</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPhysicalProperties">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Physical Properties</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCreate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>211</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listWidgetObjList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Object List</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Property</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableViewProperties">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>191</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextTerminal">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>891</width>
      <height>131</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>891</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1393</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Case"/>
    <addaction name="actionOpen_Case"/>
    <addaction name="actionSave_Case"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Case">
   <property name="text">
    <string>New Case</string>
   </property>
  </action>
  <action name="actionOpen_Case">
   <property name="text">
    <string>Open Case</string>
   </property>
  </action>
  <action name="actionSave_Case">
   <property name="text">
    <string>Save Case</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
